sdk/core/Azure.Core/src/GeoJson/GeoJsonConverter.cs

```diff
@@ -8,11 +8,10 @@
 
 namespace Azure.Core.GeoJson
 {
-#pragma warning disable AZC0014 // Dont use System.Json apis.
     /// <summary>
     /// Converts a <see cref="GeoObject"/> value from and to JSON in GeoJSON format.
     /// </summary>
-    public sealed class GeoJsonConverter : JsonConverter<GeoObject>
+    internal sealed class GeoJsonConverter : JsonConverter<GeoObject>
```

Comment:
Why are we hiding this?  We'll just rely on the attributes on each type?

Reply:
Yeah, don't need it to be public since all relevant types are marked with attributes.